src/components/final-form/BooleanField/BooleanField.tsx

```diff
@@ -12,6 +12,7 @@ export type Props = {
   position?: Responsive<Dimensions>
   label?: string
   extraLabel?: string | JSX.Element
+  extraLabelAlign?: string
```

Comment:
You could also use a literal type here: `extraLabelAlign?: "right" | "left"`

Reply:
Good idea, will do